styleguide/derek/examples/stories-interior.ejs

```diff
@@ -0,0 +1,49 @@
+<%- partial("../_partials/_ceiling") %>
+<%- partial("../_partials/_mainnav") %>
+<%- partial('../_partials/solutions/header-smallIMAC', { headline: 'Customer Stories'}) %>
```

Comment:
oh weird. That's because it's jade calling an ejs file.  Hmm.  Have you dealt with that before?  Not sure if there's an easy solution for that. I could just pull the imac banner code directly into the template instead.